Source/EasyNetQ/MessageProperties.cs

```diff
@@ -151,6 +151,18 @@ public string ContentEncoding
             set => headers = value;
         }
 
+        /// <summary>
+        ///     Copy headers
+        /// </summary>
+        public MessageProperties CopyHeaders(IDictionary<string, object> from)
+        {
+            if (from?.Count > 0)
```

Comment:
I would prefer you add { } on each statement

Reply:
done